cmd/sanitize-prow-jobs/main.go

```diff
@@ -6,36 +6,120 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
+	"sort"
 	"strings"
 	"sync"
 
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	prowconfig "k8s.io/test-infra/prow/config"
 	"sigs.k8s.io/yaml"
 )
 
-const defaultCluster = "api.ci"
-
 type options struct {
 	prowJobConfigDir string
+	configPath       string
 
 	help bool
 }
 
+type Config struct {
+	Clusters struct {
+		Default    string `json:"default"`
+		Assignment map[string]struct {
+			// Jobs override both files and groups
+			Jobs []string `json:"jobs"`
+			// Files override files
+			Files  []string `json:"files"`
+			Groups []string `json:"groups"`
+		} `json:"assignment"`
+		Groups    map[string][]string       `json:"groups"`
+		FileREMap map[string]*regexp.Regexp `json:"-"`
+	} `json:"clusters"`
+}
+
+func (config *Config) getClusterForJob(jobName string, path string) string {
+	distribution := config.Clusters.Assignment
+	clusters := make([]string, 0, len(distribution))
+	for cluster := range distribution {
+		clusters = append(clusters, cluster)
+	}
+	sort.Strings(clusters)
```

Comment:
Why?

Reply:
No need to do this with the new structure.